src/dls_bluesky_core/core/coordination.py

```diff
@@ -0,0 +1,14 @@
+import uuid
+
+
+def group_uuid(name: str) -> str:
```

Comment:
Can you make an issue for pushing this into core bluesky at some point?

Reply:
#11